packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -934,7 +933,7 @@ class InteractiveGraph extends React.Component<Props, State> {
         return point;
     };
 
-    removePoint: (arg1: Coord) => number | null | undefined = (point) => {
```

Comment:
As I'm sure you're learning now, all of these functions were used by the editor to add/remove features from the graphs. What you're trying to do would be a much bigger refactor.

Reply:
I tested the editor in storybook and didn't see any obvious problems. It seems like making these methods private shouldn't break anything, because although the `InteractiveGraphEditor` component retains a ref to the `InteractiveGraph`, it only calls the `getUserInput` method on it (https://github.com/Khan/perseus/blob/main/packages/perseus-editor/src/widgets/interactive-graph-editor.tsx#L624-L627).

Where are these `add`/`update`/`remove` methods called, outside the InteractiveGraph class itself?

Reply:
Oh, looks like I misread these errors
<img width="796" alt="Screenshot 2024-03-18 at 4 24 39 PM" src="https://github.com/Khan/perseus/assets/23404711/4e62b2b9-e04b-46f7-a212-2a9042f8a179">